azext_iot/tests/digitaltwins/test_dt_import_jobs_int.py

```diff
@@ -0,0 +1,177 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from . import DTLiveScenarioTest
+from . import generate_resource_id
+from time import sleep
+
+logger = get_logger(__name__)
+
+
+class TestDTImportJobs(DTLiveScenarioTest):
+    def __init__(self, test_case):
+        self.storage_cstring = None
+        super(TestDTImportJobs, self).__init__(test_case)
+
+    def test_dt_importjobs(self):
+        self.wait_for_capacity()
+
+        storage_account_id = self.cmd(
+            "storage account show -n '{}' -g '{}'".format(self.storage_account_name, self.rg)
+        ).get_output_in_json()["id"]
+        instance_name = generate_resource_id()
+        # @avagraw - New APIs are only supported in East US in private preview.
```

Comment:
you should make the command preview then

Reply:
If the API is private preview, then merging to dev now doesn't make sense.

Reply:
I am working with Shipra on the release plan, will update soon.

Also the above region restriction no longer exists, that is the API is available for DT instances in all regions